.github/workflows/close-stale-issues.yaml

```
@@ -0,0 +1,23 @@
+name: Close inactive issues
+on:
+  schedule:
+    - cron: "30 2 1 * *"
+
+jobs:
+  close-issues:
+    runs-on: ubuntu-22.04
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v5
+        with:
+          days-before-issue-stale: 365
+          days-before-issue-close: 365
+          stale-issue-label: "stale"
+          stale-issue-message: "This issue is stale because it has been open for a year with no activity."
+          close-issue-message: "This issue was closed because it has been inactive for a year since being marked as stale."
```

IMHO, if doing this (unsure if a good idea at all), then we should to change the language to be a bit more inviting.

E.g.

```suggestion
          close-issue-message: "This issue was closed because it has been inactive for a year since being marked as stale. Feel free to re-open this issue if it is still relevant to you."
```

And we should ensure the issue can actually be re-opened (by anyone).